### ISSUE TITLE
App crash on folders with a large number of images

### ISSUE THREAD

**Issue by sniederb** [sniederb/photocompare#1]
On any given device, folders with a large number of images can be opened, but choosing an image to start comparison results in an app crash.

```
07-21 12:21:27.772 E/ActivityManager( 2241): Transaction too large, intent: Intent { cmp=ch.want.imagecompare/.ui.compareimages.CompareImagesActivity (has extras) }, extras size: 896816, icicle size: 0
07-21 12:21:27.773 E/JavaBinder( 2241): !!! FAILED BINDER TRANSACTION !!!  (parcel size = 901120)
07-21 12:21:27.773 E/ActivityManager( 2241): Second failure launching ch.want.imagecompare/.ui.compareimages.CompareImagesActivity, giving up
07-21 12:21:27.773 E/ActivityManager( 2241): android.os.TransactionTooLargeException: data parcel size 901120 bytes
07-21 12:21:27.773 E/ActivityManager( 2241): 	at android.os.BinderProxy.transactNative(Native Method)
07-21 12:21:27.773 E/ActivityManager( 2241): 	at android.os.BinderProxy.transact(Unknown Source:86)
07-21 12:21:27.773 E/ActivityManager( 2241): 	at android.app.IApplicationThread$Stub$Proxy.scheduleTransaction(Unknown Source:28)
07-21 12:21:27.773 E/ActivityManager( 2241): 	at android.app.servertransaction.ClientTransaction.schedule(Unknown Source:2)
07-21 12:21:27.773 E/ActivityManager( 2241): 	at com.android.server.am.ClientLifecycleManager.scheduleTransaction(Unknown Source:4)
07-21 12:21:27.773 E/ActivityManager( 2241): 	at com.android.server.am.ActivityStackSupervisor.realStartActivityLocked(Unknown Source:619)
07-21 12:21:27.773 E/ActivityManager( 2241): 	at com.android.server.am.ActivityStackSupervisor.attachApplicationLocked(Unknown Source:94)
07-21 12:21:27.773 E/ActivityManager( 2241): 	at com.android.server.am.ActivityManagerService.attachApplicationLocked(Unknown Source:1233)
07-21 12:21:27.773 E/ActivityManager( 2241): 	at com.android.server.am.ActivityManagerService.attachApplication(Unknown Source:20)
07-21 12:21:27.773 E/ActivityManager( 2241): 	at android.app.IActivityManager$Stub.onTransact(Unknown Source:6041)
07-21 12:21:27.773 E/ActivityManager( 2241): 	at com.android.server.am.ActivityManagerService.onTransact(Unknown Source:119)
07-21 12:21:27.773 E/ActivityManager( 2241): 	at android.os.Binder.execTransact(Unknown Source:65)
07-21 12:21:27.773 I/ActivityManager( 2241): Process ch.want.imagecompare (pid 26988) has died: fore TOP 
```

**Comment by sniederb** [comment]
The app currently builds a list of images early on, and then passes that image list via Intent from Activity to Activity. In order to prevent a TransactionTooLargeException, this needs to be redesigned so only the image folder along with the already selected images are kept. Subsequently, each activity needs to run the image media query anew.

**Comment by sniederb** [comment]
Released 1.0.12
